qbft/spectest/tests/startinstance/empty_value.go

```diff
@@ -5,6 +5,7 @@ import (
 )
 
 // EmptyValue tests a starting an instance for an empty value (not passing value check)
+// Instance starts but with an empty value and no proposal gets broadcasted
```

Comment:
Is there a way no proposals gets broadcasted?

Reply:
@olegshmuelov I assume if CDFetcher fails fetching..